### PR TITLE
[*] Has Many Relationships - Fix sorting on hasMany fields in related data

### PR DESCRIPTION
**Pull request by DrRaider**
ForestAdmin/forestadmin#1520

### CODE REVIEW

**Comment by DrRaider** [comment]
[P1 - sort related data - sort feature is broken for reference fields](https://trello.com/c/0yp25Qf6/4031-p1-sort-related-data-sort-feature-is-broken-for-reference-fields)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Data summary/RelatedData - Fix sorting by referenced field.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: This fix description is too generic in my opinion.
You should detail in which case the sorting on belongsTo relationships was broken.

Comment (arnaudbesnier) [reply CHANGELOG.md]: I would start the description with:
```suggestion
- Has Many Relationships - ...
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Has Many Relationships - Fix sorting on hasMany fields in related data.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]:
```suggestion
- Has Many Relationships - Fix sorting on belongsTo relationship columns in related data.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: I changed my mind (Sorry for the back and forth) but maybe we should define this change as a feature for a more positive vision.
```suggestion
### Added
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+- Has Many Relationships - Fix sorting on belongsTo relationship columns in related data.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]:
```suggestion
- Has Many Relationships - Enable sorting on belongsTo relationship columns in related data.
```

**Pull request merged**